index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="author" content="Tirthrajsinh">
+  <meta name="description" content="This is Tirthrajsinh personal website for CSD112 Lab 2.">
+  <title> Tirthrajsinh's Website</title>
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <!-- Found me! -->
+  
+</head>
+<body>
+    <div class="width-wrap">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="links.html">Links</a></li>
+          <li><a href="glossary.html">Glossary</a></li>
+        </ul>
+      </nav>
+    </header>
+
+   
+    <main>
+      <h1>Welcome to Your Tirthrajsinh’s Website!</h1>
+      <p>Hello! My name is Tirthrajsinh. I am a student learning HTML and building my first personal website.</p>
+      <p>I enjoy exploring web technologies, coding, and working on creative projects.</p>
+
+      <h2>My Favourite Club</h2>
+      <img src="C:\Users\rutur\Desktop\tirthrajsinh-1\Tirthrajsinh280\images\logo.png" alt="Logo of your favorite team">
```

Comment:
```suggestion
<a href="original website of the image should be here">
      <img src="images/logo.png" alt="Logo of your favorite team">
<a>
```
- The web always uses forward slashes to denote paths.
- The original implementation is just an image and not a linked image.